dab/system.py

```diff
@@ -13,24 +18,45 @@ def restart(test_result, durationInMs=0,expectedLatencyMs=0):
     return YesNoQuestion(test_result, "Cobalt re-started?")
 
 def get(test_result, durationInMs=0,expectedLatencyMs=0):
-    dab_response_validator.validate_get_system_settings_response_schema(test_result.response)
+    try:
+        dab_response_validator.validate_get_system_settings_response_schema(test_result.response)
+    except Exception as error:
+        print("Schema error:", error)
+        return False
     response  = jsons.loads(test_result.response)
-    if response['status'] != 200:
+    request = jsons.loads(test_result.request)
+    if not EnforcementManager().is_setting_supported(request.system_setting_key):
+        if response['status'] != 501:
+            return False
+    elif response['status'] != 200:
```

Comment:
`EnforcementManager().is_setting_supported(request.system_setting_key)` fails with `AttributeError: 'dict' object has no attribute 'system_setting_key'`.

If I get [DAB 2.0 specification of `/system/settings/get`](https://github.com/device-automation-bus/dab-specification-2.0/blob/main/DAB.md#operation-getting-system-settings) right, you don't specify which setting(s) you want to get and you always retrieve _all_ settings values in the reply. You'll therefore never see a 501 response and it is pointless to check for it. I suggest reverting this change.
```suggestion
    if response['status'] != 200:
```